test.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * For local testing only. Different from actual runtime environment.
+ * 
+ * @license MIT
+ * 
+ * @author Timothy Wong (@wegylexy on GitHub)
+ */
+import { NodeVM } from "vm2";
+(async () => {
+    console.log(JSON.parse(await new NodeVM({
+        console: "off",
+        eval: false,
+        require: {
+            external: ["jsdom", "anxios"]
```

Comment:
`axios` not `anxios`

Reply:
I think the restriction is bypassed when nested.